osm_fieldwork/ODKInstance.py

```diff
@@ -50,6 +49,7 @@ def __init__(
         """
         self.data = data
         self.filespec = filespec
+        self.ignore = ["today", "start", "deviceid", "nodel", "instanceID"]
```

Comment:
Probably nodel --> model?

Reply:
ODKInstance.py is about to be deleted, as now parsers.py is used instead. Also now the tags to ignore are in the xforms.yaml file.

Reply:
well, json2osm.py is about to be deleted, now odk2osm.py handl;es all 3 formats from ODK.